int32 method
int32 sample_width
int32 sample_height
---
float64[] stdDevErrs
